assets/style/datasets.scss

```diff
@@ -0,0 +1,22 @@
+---
+# Top-level Sass files need some front matter (just this comment is enough).
```

Comment:
This file doesn't need to be Sass, try making it CSS and removing the front matter (lines 1-3).